README.develop.md

```diff
@@ -0,0 +1,52 @@
+# Development Guide
+
+## Overview
+This document guides starting development on the SARIF Viewer extension for Visual Studio Code.
```

Comment:
Rewrite: The instructions in this document will help you get started with the SARIF Viewer extension for Visual Studio Code. @jeffersonking

Reply:
Updated.